lib/susedistribution.pm

```diff
@@ -16,7 +16,7 @@ use testapi qw(send_key %cmd assert_screen check_screen check_var get_var save_s
 
 sub handle_password_prompt {
     if (!get_var("LIVETEST") && !get_var('LIVECD')) {
-        assert_screen "password-prompt";
+        assert_screen "password-prompt", get_var("S390_ZKVM") ? 120 : undef;
```

Comment:
@Soulofdestiny can you please also take a look

If we introduce these differences then I suggest to write a comment why more time is needed, at least stating a hypothesis. Could it be that s390x-zkvm (or svirt in this case) is special in this regard because we need to cover not only a switching to another tty but the whole bootup time?

I guess the problem is within `wait_boot` for svirt based tests *before* handle_password_prompt

Reply:
@okurz I have tried some investigation , pick up two failed cases at this step,  add TIMEOUT_SCALE=4 to verify running, the result is one pass and one failed. 
http://openqa.suse.de/tests/1563414#
http://openqa.suse.de/tests/1563415#
Reviewed some builds on OSD, still appear this issue randomly, So, I agree with you said. it's none with time. 
@Soulofdestiny could you have a look at this ticket. I think it need fixed at s390x backend. Thanks!